libconcurrent/bench.sh

```diff
@@ -130,7 +130,7 @@ awk 'BEGIN {debug_prefix="";
             atomics_per_op += $22;
             ops_per_cas += $24;
             i += 1} 
-     END {  time = time/i; print "\naverage time: \t", time, "";
```

Comment:
Are you sure that adding this extra \t does not break the DEBUG flag in config.h.

Reply:
I tried it and it just aligned the results of `average time` and `average througput`

Reply:
This however may have an impact in the awk script at the end bench.sh. 
A quick test is to enable the DEBUG flag in config.h file. 
After enabling it, run bench.sh and check if reports appropriate results for some of DEBUG info.

Reply:
If you think that this works ok, I think that the patch is ready to be applied.

Reply:
For completeness this is an example how it looks with DEBUG on:
```
average time: 		 32.7 
average throughput: 	 30.602 
failed cas: 	 0 
executed cas: 	 0 
successful cas:  0 
executed swap: 	 0 
executed faa: 	 1000016 
atomics: 	 1000016 
atomics per op:  0.25 
operations per cas:  inf 
```